thrust/thrust/functional.h

```diff
@@ -1241,6 +1241,12 @@ struct identity
 
 THRUST_UNARY_FUNCTOR_VOID_SPECIALIZATION(identity, THRUST_FWD(x));
 
+#ifdef _CCCL_CUDACC_BELOW_11_3
+#  define THRUST_ALIAS_ATTRIBUTE(...)
+#else
+#  define THRUST_ALIAS_ATTRIBUTE(...) __VA_ARGS__
+#endif
```

Comment:
I have needed such a macro a few times already. nvcc < 11.3 cannot parse attributes on alias declarations (like: `using T [[attribute]] = ...;`). Where is a good place to put this? Should probably go into libcudacxx or some other central place.

Reply:
As discussed offline we should put it into `libcudacxx/include/cuda/std/__cccl/attributes.h` and make it universally available as `_CCCL_ALIAS_ATTRIBUTE`